docs/sources/setup-grafana/configure-docker.md

```diff
@@ -8,45 +8,70 @@ keywords:
   - configuration
   - documentation
   - docker
+  - docker compose
 title: Configure Docker image
 weight: 1800
 ---
 
 # Configure a Grafana Docker image
 
-If you are running Grafana in a Docker image, then you configure Grafana using [environment variables]({{< relref "./configure-grafana/#override-configuration-with-environment-variables" >}}) rather than directly editing the configuration file. If you want to save your data, then you also need to designate persistent storage or bind mounts for the Grafana container.
+Running Grafana on Docker may require some additional configuration which depends on the complexity of your environment. We will explain some of those e.g. using different images, changing logging levels, defining secrets on Cloud (e.g. AWS), configuring plugins etc.
 
 > **Note:** These examples use the Grafana Enterprise docker image. You can use the Grafana Open Source edition by changing the docker image to `grafana/grafana-oss`.
 
-## Save your Grafana data
+# Supported Docker Image variants
 
-If you do not designate a location for information storage, then all your Grafana data disappears as soon as you stop your container. To save your data, you need to set up persistent storage or bind mounts for your container.
+You can install and run Grafana using the following official Docker images.
 
-### Run Grafana container with persistent storage (recommended)
+**Grafana Enterprise**: `grafana/grafana-enterprise`
 
-```bash
-# create a persistent volume for your data in /var/lib/grafana (database and plugins)
-docker volume create grafana-storage
+**Grafana Open Source**: `grafana/grafana-oss`
 
-# start grafana
-docker run -d -p 3000:3000 --name=grafana -v grafana-storage:/var/lib/grafana grafana/grafana-enterprise
-```
+Each edition is available in two variants: Alpine and Ubuntu. See below.
+
+## Alpine image (recommended)
+
+The default images are based on the popular [Alpine Linux project](http://alpinelinux.org/), available in the [Alpine official image](https://hub.docker.com/_/alpine). Alpine Linux is much smaller than most distribution base images, and thus leads to slimmer and more secure images.
+The Alpine variant is highly recommended when security is important and you want an  image size that’s as small as possible. Note that the Alpine variant uses [musl libc](http://www.musl-libc.org/) instead of [glibc and friends](http://www.etalabs.net/compare_libcs.html), so certain software might run into issues depending on the depth of their libc requirements. However, most software doesn’t have an issue with this, so this variant is usually a very safe choice.
```

Comment:
```suggestion
If you prioritize security and want to minimize the size of your image, it is recommended that you use the Alpine variant. However, it's important to note that the Alpine variant uses [musl libc](http://www.musl-libc.org/) instead of [glibc and others](http://www.etalabs.net/compare_libcs.html). As a result, some software might encounter problems depending on their libc requirements. Nonetheless, most software should not experience any issues, so the Alpine variant is generally reliable.
```